.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: Test workflow
+name: Levelup workflow
```

Comment:
Ez nem levelup workflow. Nem csak a mostani feladatnál fog lefutni, hanem a későbbiekben is. De mindegy, maradhat.